variables.tf

```diff
@@ -60,6 +60,18 @@ variable "db_port" {
   description = "Database port"
 }
 
+variable "manage_admin_user_password" {
+  type = string
+  default = false
```

Comment:
This is not correct, this should be null

Reply:
The type should be boolean, instead of string